core/new-gui/src/app/common/service/user/user.service.ts

```diff
@@ -28,13 +29,11 @@ export class UserService {
 
   public login(username: string, password: string): Observable<void> {
     // validate the credentials with backend
-    return this.authService
-      .auth(username, password)
-      .pipe(mergeMap(({ accessToken }) => this.handleAccessToken(accessToken)));
+    return this.authService.auth(username, password).pipe(map(res => this.handleAccessToken(res.accessToken)));
```

Comment:
ditto, better to keep `{ accessToken }`

Reply:
Because this switch is moved to HomeComponent.